assignment.js

```diff
@@ -31,7 +31,11 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
+let baseNumber = 2
 
+for(i = 0; i <= 12; i ++) {
+  console.log(baseNumber * i)
+}
```

Comment:
@studiosok  great job using a for loop here, code was well structured and easy to follow. An example of an alternative approach to this problem using a while loop [can be found here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L38)